packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -230,7 +234,7 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         }
     }
 
-    private slowSetup(processedOverrides: Partial<T>, deltaOptions?: DeepPartial<T>, stripSymbols = false) {
+    private slowSetup(processedOverrides: Partial<T>, deltaOptions?: DeepPartial<T> | null, stripSymbols = false) {
```

Comment:
Why the distinction between undefined and null?

Reply:
`jsonDiff` returns `null` for 'no change` cases rather than `{}`, and we need to distinguish between 'no-diff provided' and 'no-diff calculated' cases.